spark-operator/src/test/java/org/apache/spark/k8s/operator/probe/ProbeServiceTest.java

```diff
@@ -43,7 +43,6 @@
 
 import org.apache.spark.k8s.operator.metrics.healthcheck.SentinelManager;
 
-@SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
```

Comment:
We are using `UnitTestShouldIncludeAssert` instead of `JUnitTestsShouldIncludeAssert`. And, this is excluded in the config file already.